### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by Rihyx**
### Description

Please explain the changes this PR addresses here.

### Checklist

- [x] If this requires a documentation change, I have created a PR that updates the `mkdocs/docs` directory
- [x] If this requires it, I have updated the Readme
- [x] If necessary, I have updated the examples
- [x] I have verified if I need to create/update unit tests
- [x] I have verified if I need to create/update acceptance tests
- [x] If necessary, I have run acceptance tests on this branch in CI

### CODE REVIEW

File: examples/sdk-simple/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "sdk-simple",
   "version": "3.0.0",
-  "private": true,
+  "private": false,
```
Comment (mmaurello) [inline examples/sdk-simple/package.json:4]: same here, we're never supposed to publish the examples as a package

File: examples/mrl-simple/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mrl-simple",
   "version": "1.0.0",
-  "private": true,
+  "private": false,
```
Comment (Rihyx) [inline examples/mrl-simple/package.json:4]:
```suggestion
  "private": true,
```

File: examples/sdk-simple/package.json (hunk 1 above)
Comment (Rihyx) [inline examples/sdk-simple/package.json:4]:
```suggestion
  "private": true,
```

File: scripts/package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "scripts",
   "version": "1.0.0",
   "scripts": {},
-  "private": true,
+  "private": false,
```
Comment (Rihyx) [inline scripts/package.json:5]:
```suggestion
  "private": true,
```

**Review by mmaurello — APPROVED** [review]

**Pull request merged**